.github/workflows/ci-changelog.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       run:
         shell: bash -l {0}
 
+    if: github.actor != 'pre-commit-ci[bot]'
```

Comment:
Explicitly skip pull-requests generated by `pre-commit.ci`, as they don't plan to offer the ability to configure their service to allow maintainers to auto add labels